packages/react-storage/src/components/StorageManager/__tests__/StorageManager.test.tsx

```diff
@@ -13,19 +13,18 @@ import {
 } from '../types';
 import { defaultStorageManagerDisplayText } from '../utils';
 
-const storageSpy = jest
-  .spyOn(Storage, 'uploadData')
-  .mockImplementation(() => Promise.resolve({ key: 'file' }));
+const warnSpy = jest.spyOn(console, 'warn').mockImplementation();
 
-const warnSpy = jest.spyOn(getLogger('Storage'), 'warn').mockImplementation();
+const uploadDataSpy = jest.spyOn(Storage, 'uploadData');
```

Comment:
Was this changed because we're using a newer version of jest which expects different inputs or for a different reason?

Reply:
Just for better naming convention, we are spying on uploadData so changed to reflect that, as for the warnSpy that wasn't renamed, but we are spying on console instead of logger.